Fundamentals/Setup/Install/iis.md

```diff
@@ -25,6 +25,12 @@ Once that is installed you should set up a new IIS site - and make sure to add t
 For the path you want to point it at the root of your site - where the `.csproj` file is.
 :::
 
+## Add permissions to NuGet cache folder
+
+You will need to change permissions for the NuGet cache folder, which is typically `C:\users\<username>\.nuget\packages`. The user or group (IIS_IUSRS) that the IIS site is running on requires Read permissions on this folder, because this is where some of the files for Umbraco and Umbraco packages are being served from during development. When the IIS user or group does not have permission to read from the NuGet cache folder you could run into a `DirectoryNotFoundException` while running the site.
```

Comment:
```suggestion
You might need to change permissions for the NuGet cache folder - `C:\users\<username>\.nuget\packages`. The user or group (IIS_IUSRS) that the IIS site is running on requires Read permissions on this folder because this is where some of the files for Umbraco and Umbraco packages are being served from during development. If the IIS user or group does not have permission to read from the NuGet cache folder, you could run into a `DirectoryNotFoundException` while running the site.
```